Gemfile

```diff
@@ -77,7 +77,6 @@ end
 
 group :test do
   gem "selenium-webdriver"
-  gem "webdrivers"
```

Comment:
Could we not add "webdrivers" to the development group, so we can still install it on dev machines? Webdriver is always a pain to set up locally

Reply:
Good idea, done 👍

Reply:
This is working in CI now, but I need add `require "webdrivers"` for the webdrivers to work on dev. Not sure how best to do this? We could set a `CI` environment variable and require `webdrivers` unless it exists?

Reply:
Ideally we'd fix this upstream with webdrivers but I didn't want to spend too much time on it

Reply:
Could you do `require "webrivers" rescue nil` or similar?

Reply:
I tried `require "webdrivers" rescue LoadError` because LoadError isn't a StandardError but still getting: 

```
LoadError:
  cannot load such file -- webdrivers
```

Reply:
How about `require "webdrivers" if Rails.env.development?`?

Reply:
`RAILS_ENV` is set to `test` in `rails_helper`

Reply:
Oh right. Of course. Ignore me.

Reply:
I've added a commit that requires it if an environment variable doesn't exist. It feels a bit icky. The alternative is to require devs to run `brew cask install chromedriver`

Reply:
Could you not rescue from the LoadError?

Reply:
I could but not inline, it worked with the long form

```
begin 
  require "webdrivers"
rescue LoadError
  nil
end
```

Reply:
Sweet 👍

Reply:
Should I do that instead then @pezholio?

Reply:
I reckon that's the easiest thing to do, and avoids mucking around with env vars etc

Reply:
Cool, I've pushed up that change